docs/yaml/builtins/meson.yaml

```
@@ -32,6 +32,17 @@ methods:
       scripts, but for subproject scripts they have the path to the root of the
       subproject appended, usually `subprojects/<subproject-name>`.
 
+      *(since 1.4.0)* The `MESONREWRITE` environment variable contains the path
+      to the rewrite command that corresponds to the `meson` executable that
+      was used to configure the build. (This might be a different path than the
+      first executable found in `PATH`.) It can be used to remove or replace
+      any `run_command` that depends on the revision control system from the
```

```suggestion
      any [[run_command]] that depends on the revision control system from the
```
to make it into a documentation link.